webapp/src/websocket_handlers.ts

```diff
@@ -526,3 +535,37 @@ export function handleHostLowerHand(store: Store, ev: WebSocketMessage<{
         });
     }, HOST_CONTROL_NOTIFICATION_TIMEOUT);
 }
+
+export function handleHostRemoved(store: Store, ev: WebSocketMessage<{
+    call_id: string,
+    channel_id: string,
+    session_id: string,
+}>) {
+    const channelID = ev.data.channel_id;
+    const client = getCallsClient();
+    if (!client || client?.channelID !== channelID) {
+        return;
+    }
+
+    const sessionID = client.getSessionID();
+    if (ev.data.session_id !== sessionID) {
+        return;
+    }
+
+    handleCallEnd(store, {data: {channelID}} as unknown as WebSocketMessage<EmptyData>); // fake the wsMsg
+
+    if (isDesktopApp()) {
+        if (window.desktopAPI?.sendCallsError) {
+            window.desktopAPI.sendCallsError('client-error', channelID, hostRemovedMsg);
+        } else {
```

Comment:
I don't love having this in more than one place. Can we think of something else?

Reply:
Good point. Added a util function, next to the other desktop util function.